bin/package-update.sh

```diff
@@ -1,19 +1,6 @@
 #!/bin/sh
 
-# Output colorized strings
-#
-# Color codes:
-# 0 - black
-# 1 - red
-# 2 - green
-# 3 - yellow
-# 4 - blue
-# 5 - magenta
-# 6 - cian
-# 7 - white
-output() {
-	echo "$(tput setaf "$1")$2$(tput sgr0)"
-}
+. $(dirname "$0")/output.sh
```

Comment:
One of the caveats is that moving this function away also removes the documentation regarding what the color codes mean. I think it might be wise to leave it the way it is for clarity. Since there is only a single output statement in the other script how would you feel about outputting the color itself there?

Reply:
I don't like the idea of duplicating code. What I could do is adding a comment saying that `output.sh` contains the colors reference, and a comment in that file itself explaining what it does and how to reuse it.

Reply:
While I'd generally agree, I think the purpose of this `output` function is specific enough that it's fine to duplicate the code. To me this just makes the file more difficult to read if you're trying to tell what color is being output. If we're set on refactoring this function, perhaps we can offer more value and change the argument from an arbitrary integer to the color itself?

Reply:
Ok, I have "de-deduplicated" the function. This is supposed to be a temporary thing anyway.